docs/content/latest/explore/indexes-constraints/indexes-1.md

```diff
@@ -82,39 +82,23 @@ Index Scan using index_employees_department on employees (cost=0.00..5.22 rows=1
 Index Cond: (department = 'Operations'::text)
 ```
 
-For additional information, see [Create Index](https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_index/#unique).
+For additional information, see [Create index API](https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_index/#unique).
 
-## Removing Indexes
-
-You can remove one or more existing indexes using the `DROP INDEX` statement that has the following syntax:
-
-```
-DROP INDEX index_name1, index_name2, index_name3, ... ;
-```
-
-The following example shows how to remove `index_employees_department` that was created in [Creating Indexes](#creating-indexes):
-
-```sql
-DROP INDEX index_employees_department;
-```
-
-If you execute the same `SELECT` query with the `EXPLAIN` statement as in [Creating Indexes](#creating-indexes), the query plan will not include any information about the index. 
-
-## Listing Indexes
+## List indexes
 
 YSQL inherits all the functionality of the PostgeSQL `pg_indexes` view that allows you to retrieve a list of all indexes in the database as well as detailed information about every index.
 
 For details, see [pg_indexes](https://www.postgresql.org/docs/12/view-pg-indexes.html) in the PostgreSQL documentation.
 
-## Using a UNIQUE Index
+## Use a UNIQUE index
 
-If you need values in some of the columns to be unique, you can specify a B-tree index as `UNIQUE`. 
+If you need values in some of the columns to be unique, you can specify your index as `UNIQUE`. 
 
 When a `UNIQUE` index applied to two or more columns, the combined values in these columns cannot be duplicated in multiple rows. Note that since a `NULL` value is treated as distinct value, you can have multiple `NULL` values in a column with a `UNIQUE` index.
 
```

Comment:
When a `UNIQUE` index is applied

Reply:
Not sure this should be changed because the dev reviewer didnt' comment on this sentence.